python/doc/examples/data_analysis/estimate_arma.ipynb

```diff
@@ -4,22 +4,56 @@
    "cell_type": "markdown",
```

Comment:
Exhibit the expression of the ARMA illustrated ion the script with a latex formula
The list of the results at the end is not useful: don't print them!